upgrade-1.x-to-2.x.sh

```diff
@@ -740,14 +740,14 @@ BACKUPARCHIVE=/tmp/backup/newos.tar.gz
 FSARCHIVE=/mnt/data/newos.tar.gz
 
 log "Getting new OS image..."
-progress 50 "ResinOS: downloading OS update..."
+progress 50 "Downloading OS update"
 # Create container for new version
 CONTAINER=$(docker create ${IMAGE} echo export)
 if [ -z "$CONTAINER" ]; then
     log ERROR "Could not download target update image..."
 fi
 
-progress 60 "ResinOS: processing update package..."
+progress 60 "Processing OS update"
```

Comment:
maybe "processing OS update package"?

Reply:
Yeah, will update it. That will make it pretty much as long as the original was, wonder if there's any more concise way to express something similar.